pages/index.js

```diff
@@ -1,16 +1,19 @@
 import { useEffect } from 'react'
 import { useRouter } from 'next/router'
 
-const Index = () => {
+import { withGlobalStore } from 'store'
+
+const Index = ({ store }) => {
   const router = useRouter()
 
   useEffect(() => {
+    if (!store.user.organisationId) return
     const pathname = '/data-providers/[data-provider-id]'
-    const asPath = '/data-providers/1'
+    const asPath = `/data-providers/${store.user.organisationId}`
```

Comment:
It's not an organisation. It should be actual data provider ID. The relationship is one organisation to many data providers.

Reply:
To extend what I wrote here, there is a fresh [task](https://kmi-ou.atlassian.net/browse/CORE-2917) for @valerapro. Unfortunately, we are partially blocked.

Reply:
I did a mistake. I thought dataProvider is equal to organisationId